hsmodels/schemas/fields.py

```diff
@@ -948,3 +948,15 @@ def start_before_end(cls, values):
         elif end and not start:
             raise ValueError(f"A start date was not included with end date [{end}]")
         return values
+
+
+class ModelProgramFile(BaseMetadata):
+    """"""
```

Comment:
"""
    A class used to represent the metadata associated with a file used by a model program aggregation
"""